examples/material_gallery/lib/demo/colors_demo.dart

```diff
@@ -0,0 +1,133 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+const double kColorItemHeight = 48.0;
+
+class ColorSwatch {
+  const ColorSwatch({ this.name, this.colors, this.accentColors, this.threshold: 900});
+
+  final String name;
+  final Map<int, Color> colors;
+  final Map<int, Color> accentColors;
+  final int threshold;
```

Comment:
It's just the color index value beyond which titles need to be rendered white. So they look nice (and just like the spec).  I'll add comment.